library/std/src/process/tests.rs

```diff
@@ -393,6 +393,42 @@ fn test_creation_flags() {
     }
     assert!(events > 0);
 }
+/// Tests proc thread attributes by spawning a process with a custom parent process,
+/// then comparing the parent process ID with the expected parent process ID.
+#[test]
+#[cfg(windows)]
+fn test_proc_thread_attributes() {
+    use crate::os::windows::io::AsRawHandle;
+    use crate::os::windows::process::CommandExt;
+    use crate::sys::c::{DWORD_PTR, HANDLE};
+    const PROC_THREAD_ATTRIBUTE_PARENT_PROCESS: DWORD_PTR = 0x00020000;
+    let parent = Command::new("cmd.exe").spawn().unwrap();
```

Comment:
Do these processes need to be killed before the end of the test?

Reply:
They probably should be. When I ran the tests locally I did not have any lingering cmd.exe's afterwards. But, I'll add that explicitly anyway.